data/mods/TEST_DATA/items.json

```diff
@@ -1536,6 +1536,34 @@
     "vitamins": [ [ "calcium", 3 ], [ "iron", 4 ] ],
     "rot_spawn_chance": 50
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "test_whiskey_caffenated",
+    "name": { "str_sp": "caffenated whiskey" },
```

Comment:
```suggestion
    "id": "test_whiskey_caffeinated",
    "name": { "str_sp": "caffeinated whiskey" },
```

Reply:
This would break the unit test since the id would need to change there as well. Not that this data would ever be visible outside of unit tests anyway :p